src/context/fix.rs

```diff
@@ -12,7 +10,7 @@ impl Fixer {
     pub fn replace_text(&mut self, node: Node, replacement: impl Into<String>) {
         self.pending_fixes
             .get_or_insert_with(Default::default)
-            .push(PendingFix::new(node.byte_range(), replacement.into()));
+            .push(PendingFix::new(node.range(), replacement.into()));
```

Comment:
I forget what prompted this, maybe trying to mess with "only run queries against the changed parts of the tree" (which didn't work out conceptually as something I could do from "userspace" currently)?

By including this stuff you can generate `tree_sitter::InputEdit`'s and track the changes to the tree better I think is what I was doing